internal/ingress/s3sqs/s3sqs_test.go

```diff
@@ -39,8 +39,13 @@ func TestQueueReader(t *testing.T) {
 	// Range until we're done
 	var wg sync.WaitGroup
 	wg.Add(1)
-	storage.Range(func(v []byte) bool {
-		assert.Equal(t, orc, v)
+	storage.Range(func(v string) bool {
+		data, err := s3.Load(context.Background(), v)
```

Comment:
That test doesn't look right, you're returning in case of error without releasing `wg`. Also, lacks an assert on the error I think